python/main.py

```diff
@@ -20,26 +21,41 @@
     allow_methods=["GET","POST","PUT","DELETE"],
     allow_headers=["*"],
 )
-
+dbpath = pathlib.Path(__file__).parent.parent.resolve() / "db" / "mercari.sqlite3"
+db_setup_path = pathlib.Path(__file__).parent.parent.resolve() / "db" / "items.db"
```

Comment:
Nit: Usually, .db is used for database, and .sql is used for SQL files, so maybe these are a little bit confusing.